5calls/app/src/main/res/layout/issue_view.xml

```diff
@@ -8,6 +8,25 @@
         android:id="@+id/issue_name"
         style="@style/issueViewTextStyle"
         android:layout_alignParentTop="true"
+        android:layout_toStartOf="@+id/state_indicator"
+        />
+
+    <TextView
+        android:id="@+id/state_indicator"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignTop="@id/issue_name"
+        android:layout_marginStart="8dp"
+        android:background="@drawable/state_indicator_background"
+        android:paddingStart="8dp"
+        android:paddingEnd="8dp"
+        android:paddingTop="2dp"
+        android:paddingBottom="2dp"
+        android:textColor="@color/state_indicator_text"
+        android:textSize="11sp"
```

Comment:
Please put font size in dimens.xml (or better yet re-use an existing font size rather than using this, which isn't anywhere else in the app)